### PR TITLE
feat: adding parameter to customize delay in mocks

### PR DESCRIPTION
**Pull request by iorran**
## Status
**WIP**

## Description
Ability to customize the mock delay time

### CODE REVIEW

**Comment by Graveheart** [comment]
Any progress on this? It would be great if it's possible to get it merged

File: src/types/generator.ts [hunk 1]
```diff
@@ -105,6 +105,7 @@ export type GeneratorOptions = {
   override: NormalizedOverrideOutput;
   context: ContextSpecs;
   mock: boolean;
+  delay?: number;
```
Comment (anymaniax) [inline src/types/generator.ts:108]: I would add the property in `override.mock` what do you think about it?

Comment (Graveheart) [reply src/types/generator.ts:108]: I've added a PR about it
https://github.com/anymaniax/orval/pull/431

Comment (iorran) [reply src/types/generator.ts:108]: Thanks @Graveheart

**Comment by anymaniax** [comment]
The other one is merged so I close this one

**Pull request closed without merging**